TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -615,6 +615,12 @@ private bool StopDocumentBackgroundCompilation(DocumentContext contextToStop)
             return false;
         }
 
+        public string GetDefaultCopyFolder()
```

Comment:
Method should be static, it could even be turned into a static property

Reply:
Done 32ff807dc5b9cead7138edb14175dcbdd732b990